cross/tcl/Makefile

```diff
@@ -2,33 +2,74 @@ PKG_NAME = tcl
 PKG_VERS = 8.6.6
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)$(PKG_VERS)-src.$(PKG_EXT)
-PKG_DIST_SITE = http://prdownloads.sourceforge.net/tcl/
+PKG_DIST_SITE = http://prdownloads.sourceforge.net/tcl
 PKG_DIR = $(PKG_NAME)$(PKG_VERS)/unix
 
 DEPENDS =
 BUILD_DEPENDS = cross/zlib
 
 HOMEPAGE = http://www.tcl.tk/
-COMMENT  = Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language, suitable for a very wide range of uses, including web and desktop applications, networking, administration, testing and many more. Open source and business-friendly, Tcl is a mature yet evolving language that is truly cross platform, easily deployed and highly extensible.
-LICENSE  = http://tcl.tk/software/tcltk/license.html
+COMMENT = Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language, suitable for a very wide range of uses, including web and desktop applications, networking, administration, testing and many more. Open source and business-friendly, Tcl is a mature yet evolving language that is truly cross platform, easily deployed and highly extensible.
+LICENSE = http://tcl.tk/software/tcltk/license.html
 
 GNU_CONFIGURE = 1
 
-# Prevent re-declaration of fixstrtod.o
-# export ac_cv_func_strtod=yes
-# export tcl_cv_strtod_buggy=1
+TCL_PACKAGE_PATH = /var/packages/$(PKG_NAME)/target
```

Comment:
Again, this is not the correct path. Refer to $(STAGING_INSTALL_PREFIX) if anything. You could just stuff the cprrect prefix into CONFIGURE_ARGS, see my next comment.